src/NUnitEngine/nunit.engine/Services/TestAgency.cs

```diff
@@ -236,11 +244,12 @@ private ITestAgent CreateRemoteAgent(TestPackage package, int waitTime)
             {
                 Thread.Sleep( pollTime );
                 if ( !infinite ) waitTime -= pollTime;
-                ITestAgent agent = _agentData[agentId].Agent;
-                if ( agent != null )
+                var agentRecord = _agentData[agentId];
+                
+                if ( agentRecord.Agent != null )
```

Comment:
The spacing around `(` and `)` is wrong, but then again we use the same kind of spacing in the rest of the method.

Reply:
...which is why I continued to use it. 😄 

Not relevant here, but I really don't see why we should have a spacing standard. I don't think the presence or absence of a space makes the code more difficult to understand.

Reply:
But you forgot to do it 3 lines below 😄 . Personally I don't feel too strongly about a space or not, but I prefer that the code is consistent.